src/components/content/Content.js

```diff
@@ -0,0 +1,63 @@
+import React, { Fragment } from 'react';
+import Container from 'react-bootstrap/Container';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+// import Image from 'react-bootstrap/Image';
+
+import './Content.css';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+function Content() {
+  return (
+   <Fragment>
+      <Container className="Content" fluid="true">
+         <Row className="Catchword">
+            <Col>
+               <h1>Probably the best team in the world</h1>
+               <p id="P1">We gathered the team to create some fancy app while doing the React Course.</p>
```

Comment:
ok, tutaj już widzę po co było #P1 - lepiej popracować trochę wcześniej nad przygotowaniem typografii w css, to nie będziemy musieli do każdego paragrafu dodawać nowych instrukcji w css